src/app/portfolio/_components/ProjectCard.tsx

```
@@ -0,0 +1,230 @@
+          </a>
+        )}
+      </div>
+
+      {isExpanded &&
+        createPortal(
+          <div
+            className="fixed inset-0 z-50 flex items-center justify-center overflow-y-auto bg-black_opacity-60 backdrop-blur-sm scrollbar-hide"
+            onClick={handleBackdropClick}
+          >
+            <div
+              className="absolute my-10 w-[90%] max-w-[1000px] rounded-2xl bg-gray-100"
```

Biome

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)